web/yo/app/scripts/controllers/gene.js

```diff
@@ -3069,6 +3069,26 @@ angular.module('oncokbApp')
                 })
                 .finally(getSuggestedMutations);
 
+            storage.retrieveMeta().then(function(result){
```

Comment:
Need to handle the error event. If there is any issue happen to retrieveMeta method. Similar to what you did below `if(metaRealtime && metaRealtime.error) {`